src/main/kotlin/us/spaceclouds42/ekho/Ekho.kt

```diff
@@ -268,3 +280,13 @@ class ClickEventBuilder {
 fun ekho(base: String = "", method: EkhoBuilder.() -> Unit = { }): Text {
     return EkhoBuilder(LiteralText(base), method).create()
 }
+
+/**
+ * Create a [Text] object using [EkhoBuilder]
+ *
+ * @param base the first part of the text, optional, defaults to empty literal text
+ * @return a [Text] object
+ */
+fun ekhoText(base: MutableText = LiteralText(""), method: EkhoBuilder.() -> Unit = { }): Text {
```

Comment:
should accept text, not just mutable text.
names should be same, should just overload
```suggestion
fun ekho(base: Text = LiteralText(""), method: EkhoBuilder.() -> Unit = { }): Text {
```